### ISSUE TITLE
Does it support CSS injected with ApplicatioCustomizer (modern pages)?

### ISSUE THREAD

**Issue by Gennady-G** [tavikukko/Chrome-SP-Editor#57]
Hi @tavikukko ! How are You :)

I'm playing with modern pages and added small extension that adds CSS file to header. SP-Editor finds this file but does not save and approve it (I add file in Overrides tab, set "save" and "approve" checkboxes and hit ctrl+s: black "updating" message appears but no green one). Is this scenario supported with SP-Editor? If not - are we able to modify CSS files in modern UI?

Here is my extension:

```
  @override
  public onInit(): Promise<void> {

    // css url
    const css: string = "path-to-site-collection/Style Library/style.css";

    // appending in <head>:
    const head: any = document.head || document.getElementsByTagName("head")[0];
    let link: HTMLLinkElement = document.createElement("link");
    link.href = css;
    link.rel = "stylesheet";
    link.type = "text/css";
    head.insertAdjacentElement("beforeEnd", link);

    return Promise.resolve<void>();
  }
```

**Comment by tavikukko** [comment]
Hi @Gennady-G , all good here!
Could you just try to edit the css, file without using override tab? Just open the file from Sources tab, edit and save (ctrl+s).

**Comment by tavikukko** [comment]
Also, check network/console tabs for errors after trying to save the file.

**Comment by tavikukko** [comment]
fixed in latest version 5.0.4, just published.

**Comment by Gennady-G** [comment]
Thank You much @tavikukko ! :)
